mne/io/eeglab/eeglab.py

```diff
@@ -258,6 +306,24 @@ def __init__(self, input_fname, montage, preload=False, eog=(),
 
         last_samps = [eeg.pnts - 1]
         info = _get_info(eeg, montage, eog=eog)
+
+        n_chan = len(info["chs"])
```

Comment:
we don't need to do this any more, do we? don't you have `info['n_chan']` automatically computed after @wmvanvliet 's PR?